.github/workflows/lint.yml

```diff
@@ -0,0 +1,22 @@
+name: flake8 Lint
+
+on: [pull_request]
+
+jobs:
+  flake8-lint:
+    runs-on: ubuntu-latest
+    name: Lint
+    steps:
+      - name: Check out source repository
+        uses: actions/checkout@v4.1.1
+      - name: Set up Python environment
+        uses: actions/setup-python@v5.0.0
+        with:
+          python-version: "3.9"
+      - name: flake8 Lint
+        uses: py-actions/flake8@v2.3.0
+        with:
+          flake8-version: "latest"
+          update-pip: true
+          max-line-length: "100"
+          ignore: "E123,E127,E128,E221,E226,E231,E272"
```

Comment:
@dpesce Feel free to augment this list of ignores with whatever you feel is appropriate.